views/form.html

```diff
@@ -21,66 +21,125 @@ <h1 class="display-4">Onboarding Process form</h1>
-                        <input type="file" id="img" name="img" accept="image/*">
+                    <div class="from-group">
+                        <input type="file" class="form-control-file" name="fileToUpload" id="img"><em>(optional)</em>
                     </div>
-                    <!--div for image preview-->
-                    <div class="row image-preview mt-4 ml-4" id="imagePreview">
-                        <img scr="" alt="Image Preview" class="image-preview__image">
-                        <span class="image-preview__default-text">Image Preview</span>
+                    <div>
+                        <img src="../uploads/{{ @image }}" alt="Your Image">
                     </div>
+                    <div class="errors text-danger"></div>
                 </div>
-            </div><!--div main row closed-->
+            </div>
+            <!--div main row closed-->
             <div class="float-right">
-                <button type="submit" class="btn btn-primary btn-lg">Submit</button>
+                <button type="submit" class="btn btn-primary btn-lg" name="submit">Submit</button>
             </div>
+            <!--errors messages for invalid images-->
+            <check if="{{ @errors['largeImg'] }}">
+                <span class="err text-danger">{{ @errors['largeImg'] }}</span>
+            </check>
+            <check if="{{ @errors['returnCode'] }}">
+                <span class="err text-danger">{{ @errors['returnCode'] }}</span>
+            </check>
+            <check if="{{ @errors['duplicatedImage'] }}">
+                <span class="err text-danger">{{ @errors['duplicatedImage'] }}</span>
+            </check>
+            <check if="{{ @errors['wrongFileType'] }}">
+                <span class="err text-danger">{{ @errors['wrongFileType'] }}</span>
+            </check>
         </fieldset>
     </form>
-</div><!--main div closed-->
-<script src="../scripts/formScript.js"></script>
-<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
-        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+</div>
+<!--main div closed-->
+<script src="https://code.jquery.com/jquery-3.5.0.min.js"
+        integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
         crossorigin="anonymous"></script>
 <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
         integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
         crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
         integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
         crossorigin="anonymous"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="https://unpkg.com/jquery-input-mask-phone-number@1.0.9/dist/jquery-input-mask-phone-number.js"></script>
+<script>
```

Comment:
No inline JavaScript for production code. Please move this to a separate .js file